### ISSUE TITLE
Mail app fails to load after 0.15.0 upgrade

### ISSUE THREAD

**Issue by PriceChild** [nextcloud/mail#1752]
### Expected behavior
Clicking Mail app from top bar, expect to see mail accounts.

### Actual behavior
Blank white screen,
```
TypeError: "t[e] is undefined"
    o https://nextcloud.example.com/apps/mail/js/mail.js?v=3663933a-21:1
    595 https://nextcloud.example.com/apps/mail/js/6.mail.js:1
    o https://nextcloud.example.com/apps/mail/js/mail.js?v=3663933a-21:1
    o https://nextcloud.example.com/core/js/dist/main.js?v=3663933a-21:230
    H https://nextcloud.example.com/core/js/dist/main.js?v=3663933a-21:230
    u https://nextcloud.example.com/core/js/dist/main.js?v=3663933a-21:235
mail.js:79:16285
```

### Mail app

**Mail app version:** 0.15.0

**Mailserver or service:** NA

### Server configuration

**Operating system**: Ubuntu 18.04

**Web server:** Nginx

**Database:** MariaDB

**PHP version:** 7.2

**Nextcloud Version:** 16.0.0

#### Client configuration
**Browser:** Firefox 66.0.5

**Operating system:** Mac OSX

**Comment by PriceChild** [comment]
Fixed .js.map serving...

```
TypeError: "t[e] is undefined"
    Webpack 6
vue-router.esm.js:1921
    i vue-router.esm.js:1921
    p vue-router.esm.js:1963
    l vue-router.esm.js:1784
    jt vue-router.esm.js:1849
    o es6.promise.js:75
    H es6.promise.js:92
    u _microtask.js:18
```

**Comment by PriceChild** [comment]
Huh, it's suddenly started working... delayed background processing on migration?

**Comment by PriceChild** [comment]
I'm not clear then on why it failed then succeeded.

The only action taken between failing & succeeding attempts was to modify the nginx config to allow .js.map files to be served & reloading nginx. php-fpm was left alone.

**Comment by ChristophWurst** [comment]
I also ran into this now. It's indeed a problem with caching.

**Comment by PriceChild** [comment]
I think this should be closed following #1755 ?

**Comment by ChristophWurst** [comment]
Yes, thanks!
